linode/helper/framework_resource_base.go

```diff
@@ -33,6 +42,46 @@ type BaseResourceConfig struct {
 	IsEarlyAccess bool
 }
 
+type BaseResourceWithMultipleIDsImport struct {
+	BaseResource
+
+	IDNames []string
+}
```

Comment:
I'm wondering if it's better to create a new ImportState helper function to handle this case, instead of creating a new BaseResource. Because the ImportState func is the only difference, and we just need to maintain one function rather than a new base resource type.

 I also looked into our code base, and I think we can follow the way of `ImportStatePassthroughInt64ID` to build this func into `ImportStateWithMultipleIDs` and use it in related resources. For example, in the `firewalldevice` resource, we can do: 
```
func (r *Resource) ImportState(
	ctx context.Context,
	req resource.ImportStateRequest,
	resp *resource.ImportStateResponse,
) {
	resource.ImportStateWithMultipleIDs(ctx, []string{"firewall_id", "id"}, req, resp)
}
```

And the import function can be something like:
```
func (r *Resource) ImportStatePassthroughInt64ID(
	ctx context.Context,
	idNames []string,
	req resource.ImportStateRequest,
	resp *resource.ImportStateResponse,
) { // The logic }
```

Reply:
Good idea, and I think that works fine too. But one downside I can see is we will still have to define `ImportState` in every resource who will need the multiple IDs import. Having it in the base resource would further reduce code needed to future resources. But that's definitely open to discussion. What do you think?

Reply:
Just chiming in, I think I personally prefer the latter solution (helper function) from a readability perspective even thought it might be a bit more verbose.

Reply:
+1 And I feel the helper function is a bit easier to maintain. @zliang-akamai Thank you for showing us both implementation!

Reply:
@lgarber-akamai @yec-akamai, thank you both! I just pushed a commit to switch to the helper function.